### PR TITLE
Ck tile GPU verification sample develop & Add the CK TILE GEMM to the CI/CD test

### PR DESCRIPTION
**Pull request by ThomasNing**

### CODE REVIEW

**Comment by ThomasNing** [comment]
@illsilin @carlushuang @aosewski Could you take a look on this pull request on add the CK TILE verification on GPU feature and add the CK TILE GEMM to the CI/CD pipeline so we can track it in the future.

File: example/ck_tile/03_gemm/gemm_basic.cpp
Comment (illsilin) [inline example/ck_tile/03_gemm/gemm_basic.cpp:1]: I think we need to just keep a single "v" variable for verification and just use different values:
0 - no verification;
1 - verification on cpu;
2 - verification on gpu.

**Comment by ThomasNing** [comment]
@illsilin Comment Addressed. Could you take a look again?

File: Jenkinsfile
Comment (illsilin) [inline Jenkinsfile:1]: Also, here, i think, the logic is not that we want to run ck_tile FHMA tests on mi200 and mi300 in parallel to the ck_tile GEMM tests on another set of mi200 and mi300 nodes. We should make it a separate independent stage. i update that.

**Review by carlushuang — APPROVED** [review]
LGTM

**Pull request merged**